enhancements/authentication/service-ca-bundle-injection-for-admission-webhooks.md

```diff
@@ -0,0 +1,115 @@
+---
+title: service-ca-bundle-injection-for-admission-webhooks
```

Comment:
make the enhancement for injection in general

Reply:
Done